jsonnet/alertmanager.jsonnet

```diff
@@ -21,6 +21,16 @@ local authorizationRole = policyRule.new() +
                           ]) +
                           policyRule.withVerbs(['create']);
 
+local sccRole = policyRule.new() +
```

Comment:
It would make sense to have a comment IMO.

Reply:
agreed, this is a good point.